README.md

```diff
@@ -98,6 +98,16 @@ This repository contains our fork of HLSDK and restored source code for Half-Lif
 
 * Clone this repostory: `$ git clone --recursive https://github.com/FWGS/xash3d-fwgs`.
 
+#### iOS/iPadOS
```

Comment:
It would be good if you set up a CI build and deploy.

GitHub Actions provide macOS machines for free.